build.sbt

```diff
@@ -184,8 +233,8 @@ lazy val readme = (project in file("modules/readme"))
 ///////////////
 
 lazy val docs = (project in file("docs"))
-  .dependsOn(coreJVM)
-  .dependsOn(athemaJVM)
+//  .dependsOn(coreJVM)
+//  .dependsOn(athemaJVM)
```

Comment:
do we want to remove this dependency here?

Reply:
Left over from before - thanks for catching it